src/main.spec.js

```diff
@@ -90,5 +90,20 @@ test("Checks HTMLElements", () => {
       );
       expect(child.classList.contains("fs-api-entry")).toBe(true);
     }
+    if (tree.children) {
+      if (tree.previousSibling) {
+        expect(
```

Comment:
Add comment line above this `expect` statement please explaining in a human-friendly manner what you are testing here.